### PR TITLE
Fix database sid display

### PR DESCRIPTION
**Pull request by arbulu89**
# Description
Fix database sid visualization in HostList view and HanaClusterDetails view.
Before this change, the database sid was only available when the SAP system was completely registered. If the database was only there, the sid was not being displayed.
This happened because the database instance list was being obtained from the SAP system list redux state instead of its own database state

image

## How was this tested?
Tests updated

### CODE REVIEW

File: assets/js/lib/test-utils/data/databases.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+export default [
```
Comment (arbulu89) [inline assets/js/lib/test-utils/data/databases.js:1]: The databases slice default state added, as this is how it is loaded in the initial fetch (and next updates)

File: assets/js/state/selectors/cluster.js [hunk 2]
```diff
@@ -21,13 +21,17 @@ export const getClusterName =
 export const getClusterSapSystems = (clusterID) => (state) => {
   const clusterHostIDs = getClusterHostIDs(clusterID)(state);
   const {
-    sapSystemsList: { sapSystems, applicationInstances, databaseInstances },
+    sapSystemsList: { sapSystems, applicationInstances },
   } = state;
+  const {
+    databasesList: { databases, databaseInstances },
+  } = state;
```
Comment (dottorblaster) [inline assets/js/state/selectors/cluster.js:28]:
```suggestion
    sapSystemsList: { sapSystems, applicationInstances },
    databasesList: { databases, databaseInstances },
  } = state;
```

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a tiny nitpick

**Review by nelsonkopliku — APPROVED** [review]
Cool! Nice catch!

**Review by dottorblaster — APPROVED** [review]
If the CI is happy, I am

**Review by CDimonaco — APPROVED** [review]

**Pull request merged**